ast/src/expression/value.rs

```diff
@@ -24,6 +24,7 @@ pub enum ValueExpression {
     // todo: deserialize values here
     Address(#[serde(with = "crate::common::tendril_json")] StrTendril, Span),
     Boolean(#[serde(with = "crate::common::tendril_json")] StrTendril, Span),
+    Char(char, Span),
```

Comment:
Does adding this line break serialization?

Reply:
Nope `serde` supports the `char` type by default no work necessary.